ocs_ci/deployment/ocp.py

```diff
@@ -28,9 +28,10 @@ def __init__(self):
         self.deployment_type = config.ENV_DATA["deployment_type"].lower()
         if not hasattr(self, "flexy_deployment"):
             self.flexy_deployment = False
-        if not self.flexy_deployment:
+        if not self.flexy_deployment and self.deployment_platform != "ibm_cloud":
```

Comment:
Should be  `!= constants.IBMCLOUD_PLATFORM` ?

Reply:
yep, fixed